streaming/src/main/scala/org/apache/spark/streaming/receiver/ReceiverSupervisor.scala

```diff
@@ -23,9 +23,9 @@ import scala.collection.mutable.ArrayBuffer
 
 import org.apache.spark.{Logging, SparkConf}
 import org.apache.spark.storage.StreamBlockId
-import java.util.concurrent.CountDownLatch
 import scala.concurrent._
 import ExecutionContext.Implicits.global
+import java.util.concurrent.CountDownLatch
```

Comment:
The import ordering in this file was a bit of a mess to begin with, but if you're going to change it would you mind bringing it up to date with the ordering described in the style guide (e.g. java imports, then Scala, then third-party packages, then Spark)?  https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide